### PR TITLE
Update attendance patcher

### PR DESCRIPTION
**Pull request by noloerino**
Fixed a database bug where attendances were being generated for day of the section, but the API was PUTing attendances for the start of the week.

### CODE REVIEW

File: csm_web/scheduler/management/commands/export_section.py [hunk 1]
```diff
@@ -11,14 +11,12 @@ def add_arguments(self, parser):
 
     def handle(self, *args, **options):
         for email in options["emails"]:
-            profs = Profile.objects.filter(user__email=email).exclude(
-                role=Profile.STUDENT
-            )
+            profs = Mentor.objects.filter(user__email=email)
             self.stdout.write("====SECTIONS FOR MENTOR {}====".format(email))
```
Comment (Svetlitski) [inline csm_web/scheduler/management/commands/export_section.py]:
```suggestion
            self.stdout.write(f"====SECTIONS FOR MENTOR {email}====")
```
Just to be consistent with the rest of the code base

File: csm_web/scheduler/management/commands/export_section.py [hunk 2]
```diff
@@ -11,14 +11,12 @@ def add_arguments(self, parser):
 
     def handle(self, *args, **options):
         for email in options["emails"]:
-            profs = Profile.objects.filter(user__email=email).exclude(
-                role=Profile.STUDENT
-            )
+            profs = Mentor.objects.filter(user__email=email)
```
Comment (Svetlitski) [inline csm_web/scheduler/management/commands/export_section.py]:
```suggestion
            mentors = Mentor.objects.filter(user__email=email)
```

File: csm_web/scheduler/management/commands/export_section.py [hunk 3]
```diff
@@ -11,14 +11,12 @@ def add_arguments(self, parser):
...
             self.stdout.write("====SECTIONS FOR MENTOR {}====".format(email))
             for profile in profs:
```
Comment (Svetlitski) [inline csm_web/scheduler/management/commands/export_section.py]:
```suggestion
            for mentor in mentors:
```

File: csm_web/scheduler/management/commands/patchattendances.py [hunk 4]
```diff
@@ -18,14 +18,10 @@ def handle(self, *args, **options):
             week_start = today - timedelta(days=today.weekday())
```
Comment (Svetlitski) [inline csm_web/scheduler/management/commands/patchattendances.py]: This logic is incorrect. `week_start` to `week_end` will definitely be longer than a week. Say today is Sunday (day 6 in Python) 9/22, then `week_start` is `today` minus 6 days, which is 9/16. This much is correct. However we then set `week_end = today + timedelta(days=7)`, which is 9/29. Clearly 9/16 - 9/29 is longer than one week.

File: csm_web/scheduler/management/commands/patchattendances.py [hunk 5]
```diff
@@ -18,14 +18,10 @@ def handle(self, *args, **options):
...
             week_end = today + timedelta(days=7)
             for profile in students:
```
Comment (Svetlitski) [inline csm_web/scheduler/management/commands/patchattendances.py]:
```suggestion
            for student in students:
```
Don't forget to change throughout loop body

File: csm_web/scheduler/management/commands/patchattendances.py (hunk 4 above)
Comment (noloerino) [reply csm_web/scheduler/management/commands/patchattendances.py]: should've been week_start + timedelta 7, oops

**Pull request merged**